palantirnet/solr-faceted-search-react
# Tabs to spaces

This converts all the tabs to spaces... and hopefully nothing else.

To test:
- `npm run test` and observe everything passes
- review code to see that there's nothing but whitespace changes.
